examples/Console/TestOtlpExporter.cs

```diff
@@ -25,21 +25,21 @@ internal static object Run(string endpoint)
         {
             /*
              * Prerequisite to run this example:
-             * Setup an OpenTelemetry Collector to run on local docker.
+             * Set up an OpenTelemetry Collector to run on local docker.
              *
              * Open a terminal window at the examples/Console/ directory and
              * launch the OpenTelemetry Collector with an OTLP receiver, by running:
              *
              *  - On Unix based systems use:
-             *     docker run --rm -it -p 55680:55680 -v $(pwd):/cfg otel/opentelemetry-collector:0.7.0 --config=/cfg/otlp-collector-example/config.yaml
+             *     docker run --rm -it -p 55680:55680 -v $(pwd):/cfg otel/opentelemetry-collector:0.14.0 --config=/cfg/otlp-collector-example/config.yaml
```

Comment:
Thoughts on just making this `otel/opentelemetry-collector:latest`? 0.14.0 is the latest version of the collector and seems to work with this example.

Reply:
yea lets pick newest